src/Native/System.Security.Cryptography.Native/openssl.c

```diff
@@ -1098,6 +1121,15 @@ EnsureOpenSslInitialized()
     // Initialize the callback
     CRYPTO_set_locking_callback(LockingCallback);
 
+    // Initialize the random number generator seed
+    randPollResult = RAND_poll();
+    if (randPollResult < 1)
+    {
+        assert(!"RAND_poll() failed.");
```

Comment:
This causes a type coercion warning-as-error on some machines (so far, only mine...); so this should be
`assert(0 && "RAND_poll() failed.");` 

The rest were converted on Friday in https://github.com/dotnet/corefx/commit/d01a2f4d2fe33e358a83d64a42fb830012037c5a#diff-eeff6f0c4ba65023e3af722532f18575.